lib/tournament_cases.rb

```diff
@@ -0,0 +1,27 @@
+class TournamentCase < OpenStruct
+  def test_name
+    'test_%s' % description.tr(' ', '_').tr('()', '')
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def work_load
```

Comment:
the discussion about that can be found here: https://github.com/exercism/xruby/pull/463